src/test/scala/org/broadinstitute/dsde/rawls/webservice/UserApiServiceSpec.scala

```diff
@@ -229,11 +228,81 @@ class UserApiServiceSpec extends ApiServiceSpec {
           status
         }
         assertResult(Set(project1.projectName)) {
+          import org.broadinstitute.dsde.rawls.model.UserAuthJsonSupport.RawlsBillingProjectNameFormat
           responseAs[Seq[RawlsBillingProjectName]].toSet
         }
       }
   }
 
+  it should "get details of a group a user is a member of" in withTestDataApiServices { services =>
+    val group3 = RawlsGroup(RawlsGroupName("testgroupname3"), RawlsGroupEmail("testgroupname3@foo.bar"), Set[RawlsUserRef](RawlsUser(userInfo)), Set.empty[RawlsGroupRef])
+    val group2 = RawlsGroup(RawlsGroupName("testgroupname2"), RawlsGroupEmail("testgroupname3@foo.bar"), Set.empty[RawlsUserRef], Set[RawlsGroupRef](group3))
```

Comment:
Please change these email addrs so they are different, and for the below test as well